File: perfect_number.js
let number = 28
let sum = 0;

for (let index = 1; index < number; index++) {
    if (number % index == 0) {
        console.log(index);
        sum += index;
    }
}

if (sum == number) {
    console.log(number + " is a perfect number");
} else {
    console.log(number + " is not a perfect number");
}

File: perfect_number.py
number = int(input("Enter number of choice: "))
sum = 0

for index in range(1, number):
    if(number % index == 0):
        print(index)
        sum += index
if sum == number: 
    print(number, "is a perfect number")
else: 
    print(number, "is not a perfect number")
        
